test-worker/do/user/addGroup.ts

```diff
@@ -0,0 +1,33 @@
+import * as gracely from "gracely"
+import * as http from "cloudly-http"
+import { Context } from "../../Context"
+import { router } from "../../router"
+
+export async function addGroup(request: http.Request, context: Context): Promise<http.Response.Like | any> {
+	let result: gracely.Result
+	const userClient = context.do
+	const body: string[] | undefined = await request.body
+	const id = request.parameter.id
+
+	const newGroups = body || (request.parameter.group ? [request.parameter.group] : undefined)
```

Comment:
Looks good, I would prefer calling is just groups to make it shorter and easier to read (up to you though).